packages/broker/test/utils.ts

```diff
@@ -212,3 +212,16 @@ export const waitForStreamPersistedInStorageNode = async (streamId: string, part
     }
     await waitForCondition(() => isPersistent(), undefined, 1000)
 }
+
```

Comment:
These test utils could be in `test-utils` package. But if we do that test-utils would depend on `streamr-client` and Node.